appcli/orchestrators.py

```diff
@@ -314,6 +349,10 @@ def __compose_task(
             self.docker_compose_task_override_directory,
         )
 
+    def __exec_command(self, command: str) -> CompletedProcess:
```

Comment:
may not be necessary given comment above

Reply:
See above comment.